using-mobile-extensions/adobe-target/README.md

```diff
@@ -357,136 +252,116 @@ purchasedProductIds.add("125");
 
-### Clear prefetch offer cache
+**Objective-C**
 
-To clear prefetched, cached offer data, use the following:
-
-{% tabs %}
-{% tab title="Android" %}
-#### **Syntax**
+```objectivec
+NSDictionary *mboxParameters = @{@"status":@"Platinum"};
+NSDictionary *profileParameters = @{@"gender":@"female"};
 
-```java
-public static void clearPrefetchCache()
-```
+ACPTargetProduct *product = [ACPTargetProduct targetProductWithId:@"24D334" categoryId:@"Stationary"];
 
-#### **Example**
+ACPTargetOrder *order = [ACPTargetOrder targetOrderWithId:@"ADCKKBC" total:@(400.50) purchasedProductIds:@[@"34", @"125"]];
 
-```java
-Target.clearPrefetchCache();
+ACPTargetParameters *targetParameters = [ACPTargetParameters targetParametersWithParameters:mboxParameters
+profileParameters:profileParameters
+product:product
+order:order];
 ```
-{% endtab %}
-
-{% tab title="iOS" %}
-#### Syntax
+**Swift**
 
 ```objectivec
-+ (void) clearPrefetchCache;
-```
+let mboxParameters = [
+"status": "Platinum"
+]
+let profileParameters = [
+"gender": "female"
+]
 
-#### Objective-C Example
+let product = ACPTargetProduct(id: "24D334", categoryId: "Stationary")
 
-```objectivec
-[ACPTarget clearPrefetchCache];
+let order = ACPTargetOrder(id: "ADCKKBC", total: NSNumber(value: 400.50), purchasedProductIds: ["34", "125"])
+
+let targetParameters = ACPTargetParameters(parameters: mboxParameters, profileParameters: profileParameters, product: product, order: order)
 ```
 {% endtab %}
 {% endtabs %}
 
-## Visual preview <a id="visual-preview"></a>
+### Merge behavior of Target parameters
 
-Visual preview mode allows you to easily perform end-to-end QA for Target activities by enrolling and previewing these activities on your device. This mode does not require a specialized testing set up. To get started, set up a URL scheme and generate the preview links. For more information, see [Target mobile preview](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/target-mobile-preview.html).
+`TargetParameters` including `mboxParameters`, `profileParameters`, `orderParameters` and `productParameters` can be passed in Target APIs and also when creating `TargetPrefetch` or `TargetRequest` objects. The `TargetParameters` passed in public APIs are considered global parameters and are merged with the corresponding parameters in the individual `TargetRequest` or `TargetPrefetch` objects.
 
-You can also set an app deep link that can be triggered when selections are made in the preview mode by using the following methods:
+While merging, the new keys in the mbox parameters or the profile parameters are appended to the final dictionary while the keys with the same name are overwritten in each `TargetRequest` or `TargetPrefetch` object by those from global parameters. In the case or `TargetOrder` or `TargetProduct` objects, the object passed in global parameters shall replace the corresponding object in each of `TargetRequest` or `TargetPrefetch` objects.
```

Comment:
Please reword as follows:
When merging, the new keys in the mbox parameters or the profile parameters are appended to the final dictionary, and  the keys with the same name are overwritten in each `TargetRequest` or `TargetPrefetch` object by the keys from the global parameters. For `TargetOrder` or `TargetProduct` objects, the object that is passed to the global parameters replaces the corresponding object in the `TargetRequest` or `TargetPrefetch` objects."